.github/workflows/debug-apk-trace-4360.yml

```
@@ -0,0 +1,103 @@
+        run: |
+          PR_NUMBER=$(gh pr list --repo "${{ github.repository }}" --head "${{ github.ref_name }}" --json number --jq '.[0].number // empty')
+          if [ -z "$PR_NUMBER" ]; then
+            echo "No open PR for branch ${{ github.ref_name }}; skipping comment."
+            exit 0
+          fi
+          ARTIFACT_URL="${{ steps.upload_apk.outputs.artifact-url }}"
+          APK_PATH=$(find app/build/outputs/apk/debug -name "*.apk" | head -1)
+          APK_SHA=$(sha256sum "$APK_PATH" | cut -d' ' -f1)
+          gh pr comment "$PR_NUMBER" --repo "${{ github.repository }}" --body "📦 **Debug APK ready** — commit \`${{ github.sha }}\`
+
+[Download \`vultisig-4360-trace-debug-apk\`]($ARTIFACT_URL) (requires GitHub login + repo access)
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)